pkg/sshd/sshd.go

```diff
@@ -31,7 +30,7 @@ const (
 func ParseHostKeys(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interrupt) {
```

Comment:
If we are directly mounting keys via secrets why do we need ParseHostKeys? Am I missing something